src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -25,7 +25,7 @@ Copyright (c) .NET Foundation. All rights reserved.
           Hardcoded list of known implicit packges that are added to project from default SDK targets implicitly.
           Should be re-visited when multiple TFM support is added to Dependencies logic.
     -->
-    <DefaultImplicitPackages>Microsoft.NETCore.App;NETStandard.Library</DefaultImplicitPackages>
+    <DefaultImplicitPackages>Microsoft.AspNetCore.All;Microsoft.AspNetCore.App;Microsoft.NETCore.App;NETStandard.Library</DefaultImplicitPackages>
```

Comment:
@abpiskunov @davkean I'm trying to understand the consequences of changing this.

This gets passed here:
https://github.com/dotnet/sdk/blob/d6ad2456962d54b9c6c6c5ea09b0519cd6faa31a/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets#L269

And turned into a HashSet here:
https://github.com/dotnet/sdk/blob/f5db1af0f3107233ef233cd7116b5405b4047e5f/src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs#L78

And marks things as ImplicitlyDefined based on HashSet alone here:
https://github.com/dotnet/sdk/blob/f5db1af0f3107233ef233cd7116b5405b4047e5f/src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs#L171

What does dependency.IsImplicitlyDefined control in the Dependencies node? Does it impact anything else in the IDE?

It looks like this will mark explicit user package references to these packages as implicit, which feels wrong. Is that right?